### ISSUE TITLE
MXFP8 MoE training prototype graduation tracker

### ISSUE THREAD

**Issue by danielvegamyhre** [pytorch/ao#3599]
- [ ] Support emulated mode #3598 
- [ ] Fix torch reference for per group blocked layout for groups along K
- [ ] Add B200 to CI #2964 and add tests to workflow triggered by label
